articles/hooks/guides/edit-hooks-using-dashboard.md

```diff
@@ -0,0 +1,62 @@
+---
+description: How to edit Hooks using the Management Dashboard
+beta: true
+topics:
+    - hooks
+    - dashboard
+contentType: how-to
+useCase: extensibility-hooks
+v2: true
+---
+# Edit Hooks Using the Dashboard
+
+The Auth0 Management Dashboard provides a visual interface for working with Hooks. With the Dashboard, edit the code for the existing Hooks using the [Webtask Editor](https://webtask.io/docs/editor). 
```

Comment:
The Webtask editor docs are linked to three different times under this one heading - maybe a bit much?